### ISSUE TITLE
How to print learning_rate?

### ISSUE THREAD

**Issue by guotong1988** [google-research/bert#920]
How to print learning_rate so that I could see it.
Thank you!

**Comment by madhurapande19** [comment]
@guotong1988 : Any clue on how to do this?
Any help on this would be really appreciated. I too wanted to print learning rate at each step while running pretraining on TPU. Currently, it only prints loss and step. Adding learning rate to logs would help a lot. Thanks.

**Comment by guotong1988** [comment]
https://github.com/google-research/bert/issues/70
